
The usual gcc-3.4 compile fixes - including deleteing goto's ,-)

  - Rene Rebe <rene@exactcode.de>

--- gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c.orig	2004-05-13 20:42:05.535368544 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c	2004-05-13 20:42:31.767380672 +0200
@@ -135,15 +135,13 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "PilotSettings", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-pdialog init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gppd)) {
 		g_message ("gnome-pilot-pdialog init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
-
- error:
 }
 
 
--- gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c.orig	2004-05-13 20:40:51.301653784 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c	2004-05-13 20:41:29.590832944 +0200
@@ -171,18 +171,16 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "CappletMain", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-capplet init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 	
 	if (!get_widgets (gpcap)) {
 		g_message ("gnome-pilot-capplet init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 	
 	fill_widgets (gpcap);
 	init_widgets (gpcap);
-
- error:
 }
 
 GnomePilotCapplet *
--- gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c.orig	2004-05-13 20:43:06.454107488 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c	2004-05-13 20:43:43.189522856 +0200
@@ -114,15 +114,13 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "DeviceSettings", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-ddialog init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gpdd)) {
 		g_message ("gnome-pilot-ddialog init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
-
- error:
 }
 
 
--- gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c.orig	2004-05-13 20:47:56.979940848 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c	2004-05-13 20:48:22.909998880 +0200
@@ -168,19 +168,17 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", NULL, NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-druid init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gpd)) {
 		g_message ("gnome-pilot-druid init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 
 	map_widgets (gpd);
 	fill_widgets (gpd);
 	init_widgets (gpd);
-
- error:
 }
 
 
--- ./capplet/gnome-pilot-cdialog.c.orig	2004-05-13 21:18:57.751060640 +0200
+++ ./capplet/gnome-pilot-cdialog.c	2004-05-13 21:19:22.850244984 +0200
@@ -112,17 +112,15 @@
 	priv->xml = glade_xml_new (GLADEDATADIR "/gpilotd-capplet.glade", "ConduitSettings", NULL);
 	if (!priv->xml) {
 		g_message ("gnome-pilot-cdialog init(): Could not load the Glade XML file!");
-		goto error;
+		return;
 	}
 
 	if (!get_widgets (gpcd)) {
 		g_message ("gnome-pilot-cdialog init(): Could not find all widgets in the XML file!");
-		goto error;
+		return;
 	}
 	
 	gnome_dialog_close_hides (GNOME_DIALOG (priv->dialog), TRUE);
-	
- error:
 }
 
 
